### ISSUE TITLE
Failed to generate Java AccessibiltyManagerCompat.java file

### ISSUE THREAD

**Issue by Backalife** [syncfusion/Xamarin-ExpenseAnalysis#24]
The package seems missing or failing to generate a file referenced to as: AccessibilityManagerCompat_AccessibilityStateChangeListnerImplementor.java triggering an IO error.
![uikit_IOError](https://user-images.githubusercontent.com/13021926/104905539-ce7f4580-5982-11eb-910e-2603bfedfad1.jpg)

Trying to run it on Windows 10 - VSTO:
![vstoVersion](https://user-images.githubusercontent.com/13021926/104905912-49e0f700-5983-11eb-806a-625cfa4555dc.jpg)

**Comment by ghost** [comment]
Hi,
 
We have tested this application with latest Syncfusion version & Xamarin.Forms version and it is works as expected. We suspect the problem at your end is because of the application’s source location in your machine. The path may be too long or the location may not be accessible for compilation.  Could you please reduce the application path directory by moving to some other location (Example- D:\ExpenseAnalyis) and check at your end. For more details, please refer the below link.
 
[https://docs.microsoft.com/en-us/xamarin/android/troubleshooting/questions/path-too-long-exception ](https://docs.microsoft.com/en-us/xamarin/android/troubleshooting/questions/path-too-long-exception )

Please let us know if you need any further assistance in this. 
 
Regards,
Yuvaraj.
